apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/src/main/java/com/expediagroup/apiary/extensions/events/metastore/listener/KafkaMetaStoreEventListener.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.events.metastore.listener;
+
+import static com.expediagroup.apiary.extensions.events.metastore.KafkaProducerProperty.SERDE_CLASS;
+import static com.expediagroup.apiary.extensions.events.metastore.common.PropertyUtils.stringProperty;
+import static com.expediagroup.apiary.extensions.events.metastore.common.io.MetaStoreEventSerDe.serDeForClassName;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import org.apache.hadoop.conf.Configuration;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import com.expediagroup.apiary.extensions.events.metastore.common.listener.AbstractMetaStoreEventListener;
+import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableListenerEventFactory;
+import com.expediagroup.apiary.extensions.events.metastore.common.io.MetaStoreEventSerDe;
+import com.expediagroup.apiary.extensions.events.metastore.common.messaging.MessageTaskFactory;
+import com.expediagroup.apiary.extensions.events.metastore.messaging.KafkaMessageTaskFactory;
+
+public class KafkaMetaStoreEventListener extends AbstractMetaStoreEventListener {
```

Comment:
`AbstractMetaStoreEventListener` was created when we intended to have different implementations, since we now have all this code in kafka packages I'm not sure it's worth it. We could consider removing the `AbstractMetaStoreEventListener` and merging the code from it into here so it's a bit simpler and we have all the listener implementation code in one place.

Reply:
Yep, wil refactor in the next PR